q_setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Run this script to set up the qsim environment for the first time.
+# You can read the following steps to see what each is doing.
+#
+# Author: Pranith Kumar
+# Date: 01/05/2016
+# Usage: ./setup.sh {arm64}
+
```

Comment:
You need to update these comments and explain why this file is necessary. What is different from other setup files?